tests/api_test.py

```
@@ -266,3 +267,70 @@ def test_author_validation_included(self):
         check_names = {c["check"] for c in result["checks"]}
         assert "author_name" in check_names
         assert "author_email" in check_names
+
+
+class TestValidatePush:
+    """Tests for validate_push() – the programmatic push safety API."""
```

Ruff (0.15.12)

[warning] Docstring contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF002)